### PR TITLE
Rebranding

### PR DESCRIPTION
**Pull request by JesusMcCloud**
I need all eyes I can get on this, sloppy as I am. Will test integration with vclib tomorrow.
This will then be 3.5.2:

* Rebranding to Signum
  * maven coordinates: `at.asitplus.signum:$module`
  * modules
    * datatypes -> indispensable
    * datatypes-jws -> indispensable-josef
    * datatypes-cose -> indispensable-cosef
    * provider -> supreme
  * package renames
  * `crypto` -> `signum`
  * `datatypes` -> `indispensable`
  * `jws` -> `josef`
  * `cose` -> `cosef`
  * `provider` -> `supreme`

### CODE REVIEW

**Comment by whyoleg** [comment]
just curious: why? :)
BTW: nice names!

**Comment by JesusMcCloud** [comment]
> just curious: why? :) BTW: nice names!

You answered it yourself ;-)

File: README.md [hunk 1]
```diff
@@ -1,15 +1,25 @@
...
 [![GitHub license](https://img.shields.io/badge/license-Apache%20License%202.0-brightgreen.svg?style=flat)](http://www.apache.org/licenses/LICENSE-2.0)
 [![Kotlin](https://img.shields.io/badge/kotlin-multiplatform-orange.svg?logo=kotlin)](http://kotlinlang.org)
 [![Kotlin](https://img.shields.io/badge/kotlin-2.0.0-blue.svg?logo=kotlin)](http://kotlinlang.org)
 [![Java](https://img.shields.io/badge/java-17+-blue.svg?logo=OPENJDK)](https://www.oracle.com/java/technologies/downloads/#java11)
-[![Maven Central](https://img.shields.io/maven-central/v/at.asitplus.crypto/datatypes)](https://mvnrepository.com/artifact/at.asitplus.crypto/datatypes/)
+[![Maven Central](https://img.shields.io/maven-central/v/at.asitplus.signum/datatypes)](https://mvnrepository.com/artifact/at.asitplus.signum/datatypes/)
```
Comment (nodh) [inline README.md:17]: shouldn't `datatypes` be replaced with `indispensable`?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,138 @@
 # Changelog
 
+
+## 3.0
+
+### 3.5.2
```
Comment (nodh) [inline CHANGELOG.md:6]: I would jump right to `3.6.0`

**Review by nodh — APPROVED** [review]

**Review by n0900 — APPROVED** [review]

**Pull request merged**